curvefs/conf/mds.conf

```diff
@@ -64,3 +64,25 @@ mds.heartbeat.intervalMs=10000
 mds.heartbeat.misstimeoutMs=30000
 # heartbeat offline time between metaserver and mds
 mds.heartbeat.offlinetimeoutMs=1800000
+# After mds is started, a certain time delay starts to guide the metaserver to delete data
+mds.heartbeat.clean_follower_afterMs=1200000
+
+#
+# schedule config
+#
+# recoverScheduler switch
+mds.enable.recover.scheduler=false
+# RecoverScheduler round interval, the unit is second
+mds.recover.scheduler.intervalSec=5
+# Concurrency of operator on each metaserver
+mds.schduler.operator.concurrent=1
+# transfer leader timeout, after the timeout, mds removes the operator from the memory
+mds.schduler.transfer.limitSec=60
```

Comment:
This time limit is too small. Heartbeat interval is 10s.

Reply:
change it from 60 sec to 300 sec